dist/pop-up.js

```diff
@@ -5,13 +5,12 @@ const projects = [
       description:
               'A daily selection of privately personalized reads; no accounts or sign-ups required.',
       image: './img/projects/Snapshoot Portfolio.svg',
-      technologies: ['Node.js', 'Express.js', 'JWT', 'Redis'],
+      technologies: ['HTML', 'CSS', 'JavaScript',],
       liveLink: 'https://example.com',
       sourceLink: 'https://github.com/mohashyne/MSalyhu_portfolio.git',
       projectCover: './img/projects/Snapshoot Portfolio.svg',
```

Comment:
- [ OPTIONAL ] Please make sure that you fix this issue, currently, when I click the `see project` button on a project, it shows a different image on the project popup window instead of showing the same image as it is on the project card. :smile: 
- [ OPTIONAL ] Why do you have only two project cards in your project, I think it should be four projects card as it is on the Figma design :blush: